modulefiles/GDAS/hera.lua

```diff
@@ -80,6 +80,7 @@ setenv('MPIEXEC_NPROC', mpinproc)
 
 setenv('R2D2_CONFIG', '/scratch1/NCEPDEV/stmp4/Cory.R.Martin/R2D2_SHARED/config_hera.yaml')
 setenv("CRTM_FIX","/scratch1/NCEPDEV/da/Cory.R.Martin/GDASApp/fix/crtm")
+setenv("GDASAPP_TESTDATA","/scratch1/NCEPDEV/da/Cory.R.Martin/CI/GDASApp/data/")
```

Comment:
Go ahead and add `/work2/noaa/da/cmartin/CI/GDASApp/data` to the Orion modulefile. I'll rsync those paths soon.

Reply:
Added this